lib/git/config.go

```diff
@@ -15,6 +15,13 @@ import (
 	"github.com/Originate/git-town/lib/util"
 )
 
+// AddPerennialBranch adds the given branch as a perennial branch
+func AddPerennialBranch(branchName string) {
```

Comment:
We currently have the steps named in the same way. Would `AddAsPerennialBranch` make it more clear that we are not creating a new branch? I like how add/remove are opposite and would prefer to keep this and the related remove method in some form of sync. 

Our user interface uses `--add` and `--remove` and our current bash code uses `add_perennial_branch`.